### ISSUE TITLE
Add RDF

### ISSUE THREAD

**Issue by jyutzler** [opengeospatial/2D-Tile-Matrix-Set#25]
I was asked to produce some RDF for GeoPackage in conjunction with [conceptual model activity](https://github.com/opengeospatial/geopackage/issues/592). Since GeoPackage now aligns to this standard, it seemed sensible to make RDF for this standard and import it.

**Comment by joanma747** [comment]
In the previous version we had a JSON-LD version that produced some nice RDF. We were not sure there was a use case for it so we discontinued. We could revive it if there is a need to.

**Comment by jyutzler** [comment]
Yeah I saw that but I think it would be easier to start from scratch. It should go quickly.
